integration/tests/cook/test_basic.py

```diff
@@ -714,3 +714,16 @@ def test_queue_endpoint(self):
         self.assertEqual(200, queue.status_code, queue.content)
         self.assertTrue(any([job['job/uuid'] == job_uuid for job in queue.json()['normal']]))
         util.session.delete('%s/rawscheduler?job=%s' % (self.cook_url, job_uuid))
+
+    def test_basic_docker_job(self):
+        job_uuid, resp = util.submit_job(
+                self.cook_url,
+                name="check_alpine_version",
+                command="cat /etc/alpine-release",
+                container={"type": "DOCKER",
+                           "docker": { 'image': "alpine:latest" }})
+        self.assertEqual(resp.status_code, 201)
+        job = util.wait_for_job(self.cook_url, job_uuid, 'completed')
+        self.assertEqual('success', job['instances'][0]['status'])
+        self.assertEqual(False, job['disable_mea_culpa_retries'])
```

Comment:
Does this test actually care about `disable_mea_culpa_retries` or `output_url`? If not, can we kill these 2 lines?